app/components/works/available_date_component.html.erb

```diff
@@ -1,31 +1,33 @@
-<div class="mb-3 row">
-  <div class="col-sm-10 release-option">
-    <%= form.radio_button :release, 'immediate', class: 'form-check-input' %>
-    <%= form.label :release_immediate, 'Immediately once deposit is completed' %>
+<div data-controller="complex-radio">
```

Comment:
Most of the changes in this file are indents to support the addition of this controller

Reply:
:beers: for making this clear.